compose.voip.yml

```
@@ -0,0 +1,38 @@
+services:
+  freeswitch:
+    image: docker.io/rocketchat/freeswitch:latest
+    # build: .
+    restart: always
+    environment:
+      EXTERNAL_ADDRESS_FQDN: ${VOIP_DOMAIN:-}
+      FREESWITCH_PASSWORD: ${FREESWITCH_PASSWORD:-}
+      EXTENSION_PASSWORD: ${EXTENSION_PASSWORD:-}
+      EXTENSIONS: ${EXTENSIONS:-}
+      RTP_START_PORT: 20000
+      RTP_END_PORT: 20046
+      DEBUG_MODE: "true"
+      INTERNAL_INBOUND_MEDIA: "relay"
+      INTERNAL_OUTBOUND_TO_INTERNAL_MEDIA: "relay"
+      INTERNAL_OUTBOUND_TO_EXTERNAL_MEDIA: "relay"
+      EXTERNAL_INBOUND_MEDIA: "relay"
+      EXTERNAL_OUTBOUND_TO_INTERNAL_MEDIA: "relay"
+      ACCEPT_EXTERNAL_CALLS: false
+      USE_LEGACY_ARCH: false
+      DRACHTIO_DOMAIN: ${VOIP_DRACHTIO_DOMAIN}
```

```suggestion
      DRACHTIO_DOMAIN: ${VOIP_DRACHTIO_DOMAIN:-drachtio}
```
Setting this to Drachtio's container name by default, as it needs to have a value.